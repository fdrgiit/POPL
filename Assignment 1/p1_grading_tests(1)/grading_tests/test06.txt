
A02.0INTREAL

