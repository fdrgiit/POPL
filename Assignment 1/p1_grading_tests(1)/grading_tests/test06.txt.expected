2 35 A02.0INT 
2 11 REAL
